library/src/main/java/com/devbrackets/android/exomedia/core/video/ResizingTextureView.java

```diff
@@ -331,9 +338,7 @@ public void setVideoRotation(@IntRange(from = 0, to = 359) int userRotation, @In
         requestedUserRotation = userRotation;
         requestedConfigurationRotation = configurationRotation;
 
-        if (matrixManager.ready()) {
-            matrixManager.rotate(this, (userRotation + configurationRotation) % MAX_DEGREES);
-        }
+        updateRotateScale();
```

Comment:
we can't apply it when we are ready, because we haven't TextureView instance on setIntrinsicVideoSize